app/handle_msg.py

```diff
@@ -1,26 +1,59 @@
 import telebot
 import os
 from dotenv import load_dotenv, find_dotenv
+from google_trans_new import google_translator
+import google_trans_new
 
-load_dotenv(find_dotenv())
+translator = google_translator()
+language_target = ''
+text_to_translate = ''
 
-SECRET_KEY = os.environ.get("TOKEN")
-bot = telebot.TeleBot(SECRET_KEY)
+
+def get_bot_token():
+    load_dotenv(find_dotenv())
+    return os.environ.get("TOKEN")
+
+
+bot = telebot.TeleBot(get_bot_token())
+
+
+def print_all_language(message):
+    all_lang = '\n'.join([str(item) for item in google_trans_new.LANGUAGES.items()])
+    bot.send_message(message.chat.id, f"{all_lang}")
```

Comment:
```suggestion
    bot.send_message(message.chat.id, all_lang)
```
that is already string